cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageGdpr.kt

```diff
@@ -332,14 +301,4 @@ private class DataStorageGdprImpl(context: Context) : DataStorageGdpr {
             .forEach { entry -> spEditor.remove(entry.key) }
         spEditor.apply()
     }
-
-    private fun fail(param: String): Nothing = throw RuntimeException("$param not fund in local storage.")
-}
-
-internal fun DataStorageGdpr.getGDPRConsent(): Either<GDPRConsentInternal> = check {
```

Comment:
why did you delete this part? was it unused?

Reply:
Yes. There's A LOT of unused code... I think we need a separate task only to hunt those down and delete them.